input/2017-bae-europe/default_analysis_notebook.ipynb

```diff
@@ -24,15 +22,57 @@
     "jupyter_directory = os.getcwd()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## dependencies\n",
+    "* cyclus-input-gen\n",
+    "install by:\n",
+    "```\n",
+    "git clone http://github.com/arfc/cyclus-input-gen\n",
+    "cd cyclus-input-gen\n",
+    "python setup.py install\n",
+    "```"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {
-    "collapsed": true
-   },
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Usage: write_input.main([csv],[init_date], [duration], [country_list], [output_file_name])\n"
+     ]
+    }
+   ],
+   "source": [
+    "import analysis as an\n",
+    "import write_input.from_pris as fp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "import analysis as an"
+    "# write input file using write_input module\n",
```

Comment:
if the repository is called cyclus_input_gen, it's very confusing to call the main module in that repository "write_input"